espefuse.py

```diff
@@ -496,10 +498,17 @@ def dump(esp, _efuses, args):
 def summary(esp, efuses, args):
     """ Print a human-readable summary of efuse contents """
     ROW_FORMAT = "%-22s %-50s%s= %s %s %s"
-    print(ROW_FORMAT.replace("-50", "-12") % ("EFUSE_NAME", "Description", "", "[Meaningful Value]", "[Readable/Writeable]", "(Hex Value)"))
-    print("-" * 88)
+    json_efuse = {}
+    export_file=sys.stdout
+    if args.file:
+        print("Saving efuse values to " + args.file)
+        export_file = open(args.file, 'w')
+    if args.format == 'summary':
```

Comment:
This is a nitpick and I don't mind if you don't change it, but it would be a little cleaner to set a local variable like `human_output = (args.format == 'summary')` and then `if human_output` in all these tests.